src/main/resources/db/migrationv2/V2_14__added_combined_phone_number.sql

```diff
@@ -0,0 +1,3 @@
+alter table juror_mod.juror
```

Comment:
were gonna have to rename this file to go past flyway in prod - see latest number